### ISSUE TITLE
bug (InvalidCharacterError) on fieldError themeConfg property when multiple classes added to the class attribute

### ISSUE THREAD

**Issue by angelokeirsebilck** [verbb/formie#1883]
### Describe the bug

Using the themeConfig and setting the fieldError property with multiples classes breakes the property.

```twig
  fieldError: {
                            attributes: {
                                class: 'text-error text-xs mt-1'
                            }
                        },
```

![image](https://github.com/verbb/formie/assets/42614667/16b36fb7-e838-43dc-a78d-784ad7eb02a2)

The token added to the DOMTokenlist is in the wrong format and my formie error message is printed twice. When i use only one class e.g. 'text-error' it does work.

### Steps to reproduce

1. Render form with the themeConfig
2. use the fieldError property
3. set the class attribute to something with two classes ( 'text-red-500 font-medium text-sm')

### Form settings

- Multi-page form: Yes or No
- Submission Method: Ajax or Page Reload
- Client-side Validation: Yes or No
- Custom Form Templates: Yes or No

### Craft CMS version

5.1.2

### Plugin version

dev-craft-5 as 3.0.0-beta.10

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-5 as 3.0.0-beta.10"`.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.11](https://github.com/verbb/formie/releases/tag/3.0.0-beta.11)
